### ISSUE TITLE
To-do list

### ISSUE THREAD

**Issue by kibiz0r** [karan/Projects#138]
I feel like to-do lists are pretty common.

For bonus points, make it cloud-synced and shareable and support offline mode with some UI for resolving conflicts between users' changes. (Alice checks off an item while offline, Bob deletes the item, Alice comes back online... Wat do?)

**Comment by JacobGarrett** [comment]
Mute

**Comment by saulojoab** [comment]
I agree, to-do lists are super common and pretty fun to create.

Also,
> "Mute"

I laughed.

**Comment by harshbhandari7** [comment]
@kibiz0r true, now days to-do lists are considered as 'Hello World' Project for learning a new library or framework.

**Comment by veerviren** [comment]
hii
